pyproject.toml

```diff
@@ -4,7 +4,17 @@ build-backend = "flit_core.buildapi"
 
 [project]
 name = "godice"
-authors = [{name = "Particula", email = "iavtomator@gmail.com"}]
+authors = [
+    {name = "Guy Levi", email = "todo_update@it.com"},
+    {name = "Anton Ptashnik", email = "iavtomator@gmail.com"},
+]
+license = {file = "LICENSE.md"}
```

Comment:
@Geek5510 I'm not strong in licensing but I'm almost sure a published package should have a correct one. I see the repo has some custom license while open source projects usually have MIT or similar one.
Please review if the current license still fits and make changes if needed